### PR TITLE
Adding Spanish translations

### PR DESCRIPTION
**Pull request by jcconca**
Adding Spanish translations

### CODE REVIEW

**Comment by exelban** [comment]
Hi.
Thanks for the translation. Can I add you as a contributor to this translation?

**Comment by jcconca** [comment]
Yes sure.

File: Stats/Supporting Files/es.proj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+"Time" = "Tiempo";
+"Health" = "Salud";
+"Battery" = "Batería";
+"Amperage" = "Amperaje";
+"Voltage" = "Voltaje";
+"Temperature" = "Temperatura";
+"Power adapter" = "Adaptador de corriente";
+"Power" = "Corriente";
+"Is charging" = "Está cargando";
+"Time to discharge" = "Tiempo de descarga";
```
Comment (santius) [inline Stats/Supporting Files/es.proj/Localizable.strings:134]: Should be something more like "Tiempo restante"

File: Stats/Supporting Files/es.proj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+"Show removable disks" = "Mostrar los discos extraíbles";
+"Used disk memory" = "Usado %0 de %1";
+"Disk to show" = "Disco a mostrar";
+
+// Sensors
+"Temperature unit" = "Unidad de temperatura";
+
+// Network
+"Uploading" = "Cargar";
+"Downloading" = "Descargar";
```
Comment (santius) [inline Stats/Supporting Files/es.proj/Localizable.strings:109]: Should be "Descargando"

File: Stats/Supporting Files/es.proj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+// Disk
+"Show removable disks" = "Mostrar los discos extraíbles";
+"Used disk memory" = "Usado %0 de %1";
+"Disk to show" = "Disco a mostrar";
+
+// Sensors
+"Temperature unit" = "Unidad de temperatura";
+
+// Network
+"Uploading" = "Cargar";
```
Comment (santius) [inline Stats/Supporting Files/es.proj/Localizable.strings:108]: Should be "Subiendo" or "Cargando"

File: Stats/Supporting Files/es.proj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+"Health" = "Salud";
+"Battery" = "Batería";
+"Amperage" = "Amperaje";
+"Voltage" = "Voltaje";
+"Temperature" = "Temperatura";
+"Power adapter" = "Adaptador de corriente";
+"Power" = "Corriente";
+"Is charging" = "Está cargando";
+"Time to discharge" = "Tiempo de descarga";
+"Time to charge" = "Tiempo de carga";
```
Comment (santius) [inline Stats/Supporting Files/es.proj/Localizable.strings:135]: Should be "Tiempo para carga completa"

**Review by santius — CHANGES_REQUESTED** [review]
Hey - Thanks for your translation, I added some comments to certain phrases that I think are more accurate.

**Comment by jcconca** [comment]
I will update my fork with the more acurate corrections and also the oposite ones, descargando - cargando.

If i could see the app running in my computer i can do more on point corrections.

Enviado desde mi iPhone

File: Stats/Supporting Files/es.proj/Localizable.strings (hunk 3 above)
Comment (jcconca) [reply Stats/Supporting Files/es.proj/Localizable.strings:108]: I modified the file, was, wrongly created.

And I add your suggestions

File: Stats/Supporting Files/es.proj/Localizable.strings (hunk 2 above)
Comment (jcconca) [reply Stats/Supporting Files/es.proj/Localizable.strings:109]: I modified the file, was, wrongly created.

And I add your suggestions

File: Stats/Supporting Files/es.proj/Localizable.strings (hunk 1 above)
Comment (jcconca) [reply Stats/Supporting Files/es.proj/Localizable.strings:134]: I modified the file, was, wrongly created.

And I add your suggestions

File: Stats/Supporting Files/es.proj/Localizable.strings (hunk 4 above)
Comment (jcconca) [reply Stats/Supporting Files/es.proj/Localizable.strings:135]: I modified the file, was, wrongly created.

And I add your suggestions

**Review by santius — APPROVED** [review]
Approved!

**Comment by jcconca** [comment]
@exelban could you merge it?

**Comment by exelban** [comment]
Yeap, I will merge all translations before releasing the new version.

**Pull request merged**